go/vt/vtgate/evalengine/compiler_asm.go

```diff
@@ -3795,6 +3795,23 @@ func (asm *assembler) Fn_LAST_DAY() {
 	}, "FN LAST_DAY DATETIME(SP-1)")
 }
 
+func (asm *assembler) Fn_FROM_DAYS() {
+	asm.emit(func(env *ExpressionEnv) int {
+		arg := env.vm.stack[env.vm.sp-1].(*evalInt64)
+		if arg == nil {
```

Comment:
This check is not necessary / can't happen, so let's remove this. We already have emitted a nil check before we emit `FROM_DAYS` during compilation.

Reply:
got it. done.